### DISCUSSION TITLE
Impact of trees in noise reduction

### DISCUSSION THREAD

**Discussion by SantiagoRVR** [Universite-Gustave-Eiffel/NoiseModelling#712] [Q&A] [answered]
Hello everyone

I want to run the model and test the noise differences at a location when the side of the roads has trees planted vs no trees planted. I have searched through the model but it does not seem to have anything specific  for trees. As a workaround I was thinking of adding the trees as "cylindrical" buildings with a given height and append them to the buildings map. This to have an impact on the diffraction of the sound wave. Moreover, to get an impact on the ground absorption, the G value at the locations of the trees will be changed to 1. 

Is there another way to consider the impact of trees? 

Thanks in advance

**Comment by pierromond** [comment, marked as answer]
Dear Santiago, 

There is no way to consider trees implantation in NoiseModelling. But take care that contrary to what is sometimes believed, the impact of these tree lines is very low. There are many publications on the subject which I suggest you consult.

- https://www.dora.lib4ri.ch/empa/islandora/object/empa%3A5159/datastream/PDF/Wunderli-2012-An_extended_model_to_predict-%28published_version%29.pdf
-https://www.researchgate.net/publication/271601598_Using_natural_means_to_reduce_surface_transport_noise_during_propagation_outdoors
- https://www.acoustics.org.nz/sites/www.acoustics.org.nz/files/journal/pdfs/Watts,_G_NZA2011.pdf
- https://pub.dega-akustik.de/ICA2019/data/articles/000001.pdf

Kind regards.

**Reply by SantiagoRVR** [reply]
Dear, pierromond

Thank you for the information. I have taken a look at it and they are very interesting and indeed change the perspective I had about trees.

But I have another question. When ground absorption G is considered, is it taken at the maximum height of the location? meaning, If my DTM has a building with Height "x", and ground absorption "G", is this G value the ground of the building or is it the roof of the building? My question is because I want to see if there are any effects of adding soft grounds (green roofs) at the roof of buildings.

Thanks in advance

Best regards

**Reply by pierromond** [reply]
Hi @SantiagoRVR. It will be at the ground of the building. CNOSSOS-Eu is not really able to take into account green roofs... you can probably adapt the code, but you will have to do some dvpt and I am not sure it could be acoustically exact... Sorry about it !

**Reply by SantiagoRVR** [reply]
Thank you again for the response. I will change my approach accordingly.

One last question I have is. If I want to see the impact of porous pavement in a given street and I change both the G value to something other than 0 at the street location and the PVMT of the road to NL01, will this be overestimated or is it the correct approach?

**Reply by pierromond** [reply]
You can't change the G value above the street easily, but you can check in the parameters as we do for rail simulation. For the pavement, yes you can use the Netherlands pavements if you want, the list is quite vast.
![image](https://github.com/user-attachments/assets/d0f3468a-f0be-4670-afbb-9ff9cd7e0968)

**Comment by SantiagoRVR** [comment]
Thank you for this information. Do you mention that changing the G at the road is not easy because it would not be correct or because it is a difficult process to do? 

I have found a way to do the G changes quickly, but I would like to know if it is a correct approach

Thanks in advance

**Reply by pierromond** [reply]
It is strange (because you will mix pavement with grass for example), but it is quite correct I guess ;)

**Reply by SantiagoRVR** [reply]
Great. Thanks a lot for all the replies. They have been very useful.
